core/src/HIP/Kokkos_HIP_Parallel_MDRange.hpp

```diff
@@ -365,9 +365,8 @@ class ParallelReduce<CombinedFunctorReducerType,
 
   template <typename Policy, typename Functor>
   static int max_tile_size_product(const Policy&, const Functor&) {
-    using closure_type =
-        ParallelReduce<FunctorType, Kokkos::MDRangePolicy<Traits...>,
-                       ReducerType, HIP>;
+    using closure_type  = ParallelReduce<CombinedFunctorReducerType,
+                                        Kokkos::MDRangePolicy<Traits...>, HIP>;
```

Comment:
Oversight?

Reply:
Yeah, we were not forward-declaring the new interface correctly.